site/content/en/docs/Installation/Terraform/aks.md

```diff
@@ -50,6 +50,7 @@ Configurable parameters:
 - log_level - possible values: Fatal, Error, Warn, Info, Debug (default is "info")
 - cluster_name - the name of the AKS cluster (default is "agones-terraform-example")
 - agones_version - the version of agones to install (an empty string, which is the default, is the latest version from the [Helm repository](https://agones.dev/chart/stable))
+- force_update - whether or not to force the replacement/update of resource (default is true, false may be required to prevent immutability errors when updating the configuration)
```

Comment:
This is awesome!

The only thing to change is this new `force_update` section -- can we get it wrapped in a `feature` shortcode
(Example: https://agones.dev/site/docs/contribute/#within-a-page) ? Then it will only show up for the next release version, and people won't get confused when pulling down release branches.

(Might be good to put it at the end of the list, so it doesn't mess too much with the formatting)

```
{{% feature expiryVersion="1.10.0" %}}
- force_update - whether or not to force the replacement/update of resource (default is true, false may be required to prevent immutability errors when updating the configuration)
{{% /feature %}}
```

Should do the trick :+1:  (also same below)

Reply:
Done - I initially left out the short code because I would love if this could be backported to earlier versions given how small and isolated the change is, but I get the need to hide it in the meantime